worlds/metroidprime/Items.py

```diff
@@ -47,7 +47,7 @@ class MetroidPrimeItem(Item):
     "Gravity Suit": ItemData("Gravity Suit", 21, ItemClassification.progression),
     "Varia Suit": ItemData("Varia Suit", 22, ItemClassification.progression),
     "Phazon Suit": ItemData("Phazon Suit", 23, ItemClassification.progression),
-    "Energy Tank": ItemData("Energy Tank", 24, ItemClassification.useful),
+    "Energy Tank": ItemData("Energy Tank", 24, ItemClassification.useful, 14),
     "Spring Ball": ItemData("Spring Ball", 25, ItemClassification.useful),
```

Comment:
We may want to look at putting Spring Ball somewhere else (maybe 43?)  if we want to use UnknownItem1 for location checks